me.jtheoof.swappy.yaml

```diff
@@ -0,0 +1,29 @@
+id: me.jtheoof.swappy
```

Comment:
`jtheoof.me` doesn't seem to have a website. it's hard to verify it's actually valid.

Reply:
Its used in upstream: https://github.com/jtheoof/swappy/blob/596b9a8b17234303a9d4d2ea1d3107cb5442253e/src/application.c#L948

Should I use `io.github.jtheoof.swappy` instead?

Reply:
then you should ask them at the same time as you contact them.